Script/version.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+if which jq >/dev/null; then
+    echo "jq is installed"
+else
+    echo "error: jq not installed.(brew install jq)"
+fi
+
+NC='\033[0m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+
+echo "Target version - ${BLUE}$1${NC}"
+echo "-----------------AEPEdge------------------"
+PODSPEC_VERSION_IN_AEPEdge=$(pod ipc spec AEPEdge.podspec | jq '.version' | tr -d '"')
+echo "Local podspec version - ${BLUE}${PODSPEC_VERSION_IN_AEPEdge}${NC}"
+# SOUCE_CODE_VERSION_IN_AEPEdge=$(cat ./AEPSignal/Sources/SignalConstants.swift | egrep '\s*EXTENSION_VERSION\s*=\s*\"(.*)\"' | ruby -e "puts gets.scan(/\"(.*)\"/)[0] " | tr -d '"')
+# echo "Souce code version - ${BLUE}${SOUCE_CODE_VERSION_IN_AEPEdge}${NC}"
+
+# if [[ "$1" == "$PODSPEC_VERSION_IN_AEPEdge" ]] && [[ "$1" == "$SOUCE_CODE_VERSION_IN_AEPEdge" ]]; then
+if [[ "$1" == "$PODSPEC_VERSION_IN_AEPEdge" ]]; then
```

Comment:
https://github.com/adobe/aepsdk-core-ios/blob/c9abdd670e602c515889a94b189dab6b18887291/Script/version.sh#L43 
Should we also check the version in source code?